app/src/main/java/ch/sdp/vibester/activity/LyricsBelongGameActivity.kt

```diff
@@ -20,6 +21,7 @@ private const val REQUEST_AUDIO = 100
 
 class LyricsBelongGameActivity : AppCompatActivity() {
```

Comment:
What does the classs represent ? Doc ?

Reply:
It's my task for this week, I will add it in my PR